From: "brian m. carlson" <sandals@crustytoothpaste.ath.cx>
Subject: quilt: breaks when POSIXLY_CORRECT=1
Date: Fri, 25 Jan 2008 20:02:28 +0000
Closes: #462578

quilt breaks when POSIXLY_CORRECT is set to 1, because it invokes
patch, and patch does not work non-interactively in POSIX mode.  (I
believe this is the mandated behavior, unfortunate as it is.) If you
want to rely on non-POSIX semantics of patch, you should unset
POSIXLY_CORRECT (and probably _POSIX2_VERSION and POSIXLY_PEDANTIC as
well).


---
 bin/quilt.in |    4 ++++
 1 file changed, 4 insertions(+)

Index: b/bin/quilt.in
===================================================================
--- a/bin/quilt.in
+++ b/bin/quilt.in
@@ -6,6 +6,10 @@
 #
 #  See the COPYING and AUTHORS files for more details.
 
+# unset posix strict conformance variables since patch cannot be run
+# non-interactively when they are set (doh!).
+unset POSIXLY_CORRECT _POSIX2_VERSION POSIXLY_PEDANTIC
+
 export TEXTDOMAIN=quilt
 export TEXTDOMAINDIR=@LOCALEDIR@
 
